### PR TITLE
Add win10toast recipe

### PR DESCRIPTION
**Pull request by setu4993**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by setu4993** [comment]
@conda-forge/staged-recipes : This PR is ready for review.

File: recipes/win10toast/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/d4/ba/95c0ea87d9bcad68b90d8cb130a313b939c88d8338a2fed7c11eaee972fe/win10toast-0.9-py2.py3-none-any.whl
+  sha256: 44e5afa1001de88a0ee533872231521fa67c7d144f39974089af242d9c4620a4
+
+build:
+  skip: True  # [not win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install win10toast-0.9-py2.py3-none-any.whl"
```
Comment (scopatz) [inline recipes/win10toast/meta.yaml]: Please install from source, rather than from a wheel

File: recipes/win10toast/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
```
Comment (scopatz) [inline recipes/win10toast/meta.yaml]: Only pip and python are needed here

File: recipes/win10toast/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{% set name = "win10toast" %}
+{% set version = "0.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/jithurjacob/Windows-10-Toast-Notifications.git
```
Comment (scopatz) [inline recipes/win10toast/meta.yaml]: Please install from a tarball rather than a repo

**Comment by setu4993** [comment]
@scopatz : Thanks for taking a look at this and the recommendations. I cannot find a tarball for this package on [PyPi](https://pypi.org/project/win10toast/0.9/#files), which is why I went the wheel way, and am now trying to get the github source working.

It is quite an outdated package that hasn't been updated in >1 year and some recent PRs for fixes at source are not updated.

File: recipes/win10toast/meta.yaml (hunk 1 above)
Comment (setu4993) [reply recipes/win10toast/meta.yaml]: Using a git source now.

File: recipes/win10toast/meta.yaml (hunk 3 above)
Comment (setu4993) [reply recipes/win10toast/meta.yaml]: Sadly, there is no tarball available for this project (on PyPi or on GitHub releases).

Comment (scopatz) [reply recipes/win10toast/meta.yaml]: Github will generate a release for you if it is tagged though

Comment (setu4993) [reply recipes/win10toast/meta.yaml]: There are no tags on the repo either :/.

File: recipes/win10toast/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
+{% set name = "win10toast" %}
+{% set version = "0.9" %}
```
Comment (jakirkham) [inline recipes/win10toast/meta.yaml]: How was this version determined?

File: recipes/win10toast/meta.yaml (hunk 3 above)
Comment (scopatz) [reply recipes/win10toast/meta.yaml]: Hmmm I think we need to encourage the devs to release an actual source tarball, either with `twine sdist` or by git tag

File: recipes/win10toast/meta.yaml (hunk 4 above)
Comment (setu4993) [reply recipes/win10toast/meta.yaml]: PyPI: https://pypi.org/project/win10toast/0.9/#files

File: recipes/win10toast/meta.yaml (hunk 3 above)
Comment (setu4993) [reply recipes/win10toast/meta.yaml]: Agree, but as I mentioned in a [comment](https://github.com/conda-forge/staged-recipes/pull/11051#issuecomment-599274337), it is a package that hasn't been updated in a while even for simple fixes (one of which required the patch), and has multiple open, unresponded PRs. The absence of this recipe was blocking a version update on [another project](https://github.com/conda-forge/knockknock-feedstock/pull/4) that I contribute to

File: recipes/win10toast/meta.yaml (hunk 4 above)
Comment (setu4993) [reply recipes/win10toast/meta.yaml]: And the [source](https://github.com/jithurjacob/Windows-10-Toast-Notifications/blob/9d52b73f1af6c60162cf09b99269c4f7b13cdb00/setup.py#L23).

File: recipes/win10toast/meta.yaml (hunk 3 above)
Comment (npavlovikj) [reply recipes/win10toast/meta.yaml]: I agree with Anthony that the developers should be encouraged to release an official tarball, so opening an issue regarding wouldn't hurt. 

Looks like the PyPI wheel was updated on 01/25/2018 which corresponds with commit "9d52b73f1af6c60162cf09b99269c4f7b13cdb00", so maybe `url: https://github.com/jithurjacob/Windows-10-Toast-Notifications/archive/9d52b73.tar.gz` can be used in the meantime ?

Comment (isuruf) [reply recipes/win10toast/meta.yaml]: If there's no tag in github, I don't see an issue with using the wheel.

Comment (setu4993) [reply recipes/win10toast/meta.yaml]: Yeah, that's why I used a wheel at start. Happy to use the tarball from the latest commit and opening an issue at source requesting a tag.

Comment (setu4993) [reply recipes/win10toast/meta.yaml]: Switched source to the commit tarball, opened an [issue](https://github.com/jithurjacob/Windows-10-Toast-Notifications/issues/66) at source.

**Comment by setu4993** [comment]
Thanks for reviewing and approving, @scopatz!

**Comment by setu4993** [comment]
@conda-forge/staged-recipes : Bump, can someone please merge this?

**Comment by setu4993** [comment]
@conda-forge/staged-recipes : Bump, this is ready to merge.

**Pull request merged**